### ISSUE TITLE
Minifying not working

### ISSUE THREAD

**Issue by iamsoaw** [rkingon/Craft-Plugin--Sprockets#2]
Hello. Minifying not working, it seems some problem with submodule vendor/matthiasmullie/minify, it does not exists.

**Comment by rkingon** [comment]
Indeed. 

For now, you can grab that here: https://github.com/matthiasmullie/minify

I would honestly not even use this plugin anymore- I have a much better time using grunt.

**Comment by iamsoaw** [comment]
Do you use grunt right from Craft CMS (as a plugin), or run it manually/from your editor?

**Comment by rkingon** [comment]
i use it in the cli, "grunt watch" to look for chages as i am dev'ing. 

you can also use a plugin like: https://github.com/hollandben/grunt-cache-bust to handle asset hash.

there is another which output's a json file of what the asset hash's are, i was going to write a little twig extension to read that json file so you can use a craft tag to output your css/js and it'll automatically handle the naming convention

**Comment by iamsoaw** [comment]
Yes, these tools are great, and I love Node.js, but they are external. I believe that the power of Craft is in its integrity, everything is inside the core and we rely on PHP functions and modules. That's why I like your plugin over any other external tools.

PS: Please check your Facebook, I have sent you a message regarding this plugin (not a bugreport).
